src/svg/Painter.ts

```diff
@@ -193,7 +198,8 @@ class SVGPainter implements PainterBase {
         return vNodeToString(this.renderToVNode({
             animation: defaultOpts.cssAnimation,
             willUpdate: false,
-            compress: true
+            compress: true,
+            useViewBox: opts.useViewBox
```

Comment:
One question: Can it be better if `useViewBox` default to `true`?

Reply:
Make sense. Updated.